Assets/MRTK/Core/Utilities/StandardShader/ClippingPrimitive.cs

```diff
@@ -141,6 +143,14 @@ public IEnumerable<Renderer> GetRenderersCopy()
             return new List<Renderer>(renderers);
         }
 
+        /// <summary>
+        /// Checks is renderer is being clipped by this clipping primitive.
+        /// </summary>
+        public bool CheckIfClipped(Renderer renderer)
```

Comment:
Is this being used anywhere? Maybe a good name would be ContainsRenderer? Or something akin to "this primitive currently influences this renderer. Just because "Check If Clipped" might be interpreted as "is this renderer straddling the clip vs not-clipped threshold?"

Reply:
I added this to check renderers instead of making a copy of the renderers on every frame from scroll. But currently not using it. Can delete it or rename it.

Reply:
I'll leave it up to you! Just was curious why it was added. It might be a good API to have for developers (or we might not want to bother supporting it).

Reply:
Just a heads up I don't think this was removed.